css/style.css

```diff
@@ -18,7 +18,7 @@ html {
     font-size: 4rem;
     margin-bottom: 0;
   }
-  
+  /* min-width? */  
```

Comment:
Consider using 'min-width' in place of 'max-width'. This is a best practice for a modern Mobile-First approach to development.

Reply:
- [ ] I think it would be better to rephrase to sound like a suggestion instead of order 👍

For example:
```
This is a polite suggestion: "I think it would be better to use `min-width` in place of `max-width`. This is a best practice for a modern Mobile-First approach to development. 🙏 "
```

Reply:
Hello @TuyishimireEric. I updated my comment. What do you think? 🤔